contributors/guide/reviewer-guidelines.md

```diff
@@ -0,0 +1,71 @@
+
+### A Patch/Minor Chage
+- If the PR is a patch or a minor change in the repository, You can browse through the changes line of codes through the GitHub UI. You can use the `Split Diff View` for easy view of each LOC and also you can see changes made in the each commit of the patch using the UI.
+
+### Major Change:
+- If the PR is a major change which may affect a lot of section of your repository, You can pull the local changes of the PR into your local system and browse through each and every change made in the PR. You can also test & play around with the changes done in the PR to ensure each and every change is as expected.
+- To fetch the Pull Request locally, Use this -
+```sh
+$ git fetch origin pull/ID/head:BRANCHNAME
+$ git checkout BRANCHNAME
+```
+
+## Reviewing Ideology
+
+### Deciding what to review ?
+- As a Maintainer, you will might be completely overloaded with the Pending PRs left to review. Some of which will be from experienced contributors and some from the new contributor.
+- Since you know the experienced contributor already knows the guidelines for PR and Code to be written for repository. You should review the PR and patches from the experienced contributors first, as it will be less time consuming.
+- Until you have a large block of free time, You must put the newcomer PR for hold. Afterwhich, you should go line by line to the code changes in the PR and please avoid to criticse the `Code Structure`, `Coding Style` of the newcomer's PR.
+
+### Assigned Reviews
+- Maintainers can assign reviews to other maintainers, when appropriate. The
+assignee becomes the shepherd for that PR and is responsible for merging the PR
+once they are satisfied with it or else closing it. The assignee might request
+reviews from non-maintainers.
+
+### What after succesfull review ?
+- If the code changes in the PR looks good, You can trigger the PR Builder to test the PR for tests defined.
+    + PR Builder can be triggered by a reply of `/ok-to-test` by a **Kubernetes Member**
+- A PR is considered to be `Ready To Merge` if it matches the `Tide Conditions` after which `Tide` can successfully merge the PR. You can find the set of conditions [here](https://prow.k8s.io/tide).
+
+## Something to Avoid
+
+- From newcomer's perspective, If the PR is ignored for days and afterwhich it receives a long list of `Review Comments` specifying the changes and mistakes in their PR may feel too very harsh.
+- For to avoid such problems, Here are some general guidelines to avoid while reviewing the PRs :
+    + Avoid stating your personal opinion as a fact in the PR's comments.
+    + Avoid avalanche of comments and judgemental questions.
+    + Don't ask the developer to fix the problem which is not covered in the following Issue Description or PR.
+    + Avoid using sarcastic comments and try to be formal & precise.
+
+## Additional Readings
```

Comment:
Another really good resource is Tim's presentation from this past contributor summit -- it has some invaluable tips for reviewing prs

[Keeping the Bar High - How to be a bad ass Code Reviewer](https://youtu.be/OZVv7-o8i40)